tools/MultiProjectValidator/AnalysisRules/DependencyMismatch/DependencyInfo.cs

```diff
@@ -9,7 +10,7 @@ public static DependencyInfo Create(ProjectContext context, LibraryDescription l
             return new DependencyInfo
             {
                 ProjectPath = context.ProjectFile.ProjectFilePath,
-                Version = library.Identity.Version.ToString(),
+                Version = library.Identity.Version?.ToString() ?? String.Empty,
```

Comment:
This change looks fine to me, but I am curious in what situations will we see versionless deps?